test/unit/presenters/dashboard/received_messages_presenter_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class UnreadMessagesPresenterTest < Draper::TestCase
```

Comment:
This is the old `unread_messages_presenter_test.rb`. Just rewrote it in a way it's more standard, I would say. Using Mocha to mock instead of creating a custom `MessagesDouble` like before.